### PR TITLE
Create card component

### PR DESCRIPTION
**Pull request by Szulerinio**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-  [textColor]="textColor"
-  [backgroundColor]="backgroundColor"
-  [wide]="wide"
-  [disabled]="disabled"
-  (buttonClicked)="temporaryLogger('button1')"
-  [value]="'draw'"
-  [translationParameters]="{ name: name === '' ? defaultName : name }"
->
-</app-button>
+<div [class]="'container'">
```
Comment (radKoz) [inline src/app/app.component.html:1]: `class="container"` without brackets is enough if you just want to add css class without any logic

File: src/app/shared/components/card/card.component.html [hunk 2]
```diff
@@ -1 +1,3 @@
-<p>card works!</p>
+<div [class]="'container'">
```
Comment (radKoz) [inline src/app/shared/components/card/card.component.html:1]: same here

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+  <app-card>
+    <app-button
+      [textColor]="textColor"
+      [backgroundColor]="backgroundColor"
+      [wide]="wide"
+      [disabled]="disabled"
+      (buttonClicked)="temporaryLogger('button1')"
+      [value]="'draw'"
```
Comment (radKoz) [inline src/app/app.component.html:9]: I wonder what is this value for? where this `draw` is used?

File: src/app/app.component.html [hunk 4]
```diff
@@ -1,23 +1,26 @@
...
+      [translationParameters]="{ name: name === '' ? defaultName : name }"
```
Comment (danieljanusm) [inline src/app/app.component.html:10]: what if name would be undefined?

`{ name: !name?.length ? defaultName : name }`

File: src/app/app.component.html [hunk 5]
```diff
@@ -1,23 +1,26 @@
...
-<app-card></app-card>
-<app-number-picker></app-number-picker>
-<app-select></app-select>
-<app-input></app-input>
+    <app-button
+      [wide]="!wide"
+      [disabled]="!disabled"
+      (buttonClicked)="temporaryLogger('button2')"
+      [value]="'draw'"
+      [translationParameters]="{ name: name === '' ? defaultName : name }"
```
Comment (danieljanusm) [inline src/app/app.component.html:19]: same here

File: src/app/app.component.html (hunk 4 above)
Comment (Szulerinio) [reply src/app/app.component.html:10]: a?.length will be undefined and !undefined evaluates to true, so AFAIK it should not cause problems

File: src/app/shared/components/card/card.component.scss [hunk 6]
```diff
@@ -0,0 +1,12 @@
+@import "variables";
+.container {
+  background-color: $white;
+  padding: 50px 60px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 7px;
+  box-shadow: #0002 0 0 5px;
```
[the remark this answers is not in the document]
Comment (Szulerinio) [reply src/app/shared/components/card/card.component.scss:8]: It is black that is 2/15 opaque - same as #00000022 or rgba(0,0,0,1.333) 
I don't know if it's plausible to make half-transparent shadow of color based on other variable and it isn't used anywhere else in this app so I decided to make it local value only

**Review by ogigg — APPROVED** [review]

**Pull request merged**